### ISSUE TITLE
"Node Launcher" does not Launch on Mac :(

### ISSUE THREAD

**Issue by nk1tz** [lightning-power-users/node-launcher#213]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request

* **What is the current behavior?**
After unzipping the Mac app, attempting to launch it fails immediately.
The app icon appears momentarily in the dock, and then disappears almost immediately.
If activity monitor is open you can see the process appear and disappear at the same time.

* **If the current behavior is a bug, please provide the steps to reproduce**

1. Download the v5.6.0 **macOS.Node.Launcher.app.zip**.
2. Unzip the file. It should appear as "Node Launcher" with the lightning icon in the same folder.
3. Run this unzipped "Node Launcher" file.
4. Notice that the app icon appears in MacOs dock momentarily before disappearing. 

* **What is the expected behavior?**
That the app runs and become useable.

* **What is the motivation / use case for changing the behavior?**
To allow easy participation in the lightning network.

* **Please tell us about your environment:**

  - Operating System: macOS El Capitan 10.11.6
  - 8 GB ram
  - 500 GB HDD
  - MacBookPro 13" mid-2012.

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by jasonzhouu** [comment]
The same problem happened on my Macbook Air 13" Early 2015 ( macOS Mojave 10.14.2)
After I launch the node launcher, the icon in the dock jump for a while, then keep quiet for a while, then exit totally, and can't be launcher anymore.
After it, `~/Library/Application Support/Node Launcher` appeared:
```
$ tree .
.
├── bin
│   ├── bitcoin-cli
│   ├── bitcoin-qt
│   ├── bitcoin-tx
│   ├── bitcoind
│   └── test_bitcoin
├── bitcoin
│   ├── bitcoin-0.17.1
│   │   ├── bin
│   │   │   ├── bitcoin-cli
│   │   │   ├── bitcoin-qt
│   │   │   ├── bitcoin-tx
│   │   │   ├── bitcoind
│   │   │   └── test_bitcoin
│   │   ├── include
│   │   │   └── bitcoinconsensus.h
│   │   ├── lib
│   │   │   ├── libbitcoinconsensus.0.dylib
│   │   │   └── libbitcoinconsensus.dylib -> libbitcoinconsensus.0.dylib
│   │   └── share
│   │       └── man
│   │           └── man1
│   │               ├── bitcoin-cli.1
│   │               ├── bitcoin-qt.1
│   │               ├── bitcoin-tx.1
│   │               └── bitcoind.1
│   └── bitcoin-0.17.1-osx64.tar.gz
└── lnd
    ├── lnd-darwin-amd64-v0.5.2-beta
    └── lnd-darwin-amd64-v0.5.2-beta.tar.gz
```

**Comment by mguttentag** [comment]
I was having similar issues (Node Launcher bouncing in dock then disappearing + not starting) and appear to have solved it! I should note that previously I was also having an issue getting Bitcoin Core to open on its own, it would display, "Error: Cannot parse configuration file: specified data directory..." -- but it was opening fine through Node Launcher, at least until I restarted my Mac and then Node Launcher wouldn't boot. Anyway, it's working now and here's what I did in case it's helpful:

**tl;dr: I moved my Bitcoin datadir somewhere else (outside of "Library") then modified the .conf file and bitcoin settings to point to the new location, then everything worked again.**

1) Moved my "bitcoin" data directory out of the Library (I moved it from `Users/**/Library/Application\ Support/Bitcoin` over to `Users/**/Bitcoin`)
2) Edited Bitcoin.conf so data directory read the new location of Bitcoin folder (`Users/**/Bitcoin`)
_[here I  also used Time Machine to restore an earlier version of Bitcoin-Qt, i.e. from a few days ago, but I do not think that was necessary. Same with Node Launcher, but they were otherwise the same versions of the program]_ 
3) Opened Terminal.app and ran `/Applications/Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt --resetguisettings` 
4) When the "Welcome to Bitcoin Core" dialogue came up, I clicked "Use a custom data directory" and directed it to the new location of the Bitcoin folder (`Users/**/Bitcoin`). 
5) Then Bitcoin Core seemed to be working, so I quit it. I then launched Node Launcher. Node Launcher now loaded. When I pressed "Launch Bitcoin" in Node Launcher, it launched, and LND worked too. 

Let me know if that works for your problem too!

**Comment by locky4hope** [comment]
I'm getting the exact same errors and am running same Mac OS X 10.11.6.

30/04/2019 09:41:51.417 launchservicesd[77]: SecTaskLoadEntitlements failed error=22
30/04/2019 09:41:51.423 launchservicesd[77]: SecTaskLoadEntitlements failed error=22
30/04/2019 09:41:51.425 launchservicesd[77]: SecTaskLoadEntitlements failed error=22
30/04/2019 09:41:51.436 appleeventsd[49]: SecTaskLoadEntitlements failed error=22
30/04/2019 09:41:51.510 com.apple.xpc.launchd[1]: (Node Launcher.64032[787]) Service exited with abnormal code: 255

**Comment by PierreRochard** [comment]
@nk1tz  Two things to try: following the developer instructions here https://github.com/lightning-power-users/node-launcher#development or updating to a newer version of macOS

I think you're getting an error because I build the Node Launcher on a newer version of macOS 

@jasonzhouu I think you're having a separate problem, does the latest release work? https://github.com/lightning-power-users/node-launcher/releases

**Comment by LAMike310** [comment]
Updating to python 3.7 worked for me, I now see the icon when I restarted and installed
